Lib/platform.py

```diff
@@ -329,20 +333,31 @@ def _linux_distribution(distname, version, id, supported_dists,
 
     """
     try:
-        etc = os.listdir(_UNIXCONFDIR)
-    except OSError:
-        # Probably not a Unix system
+        _os_release = open(_UNIXCONFDIR)
+        _os_release_content = _os_release.read()
+        get_name = re.findall(r'NAME=.*', _os_release_content)[0]
```

Comment:
Why not just `re.search()` if you use only the first found match? What if the patter will be not found in the file?

The pattern `r'NAME=.*'` gives false positive for e.g. `VERSION_CODENAME=`.